browser/about_flags.cc

```diff
@@ -42,6 +42,10 @@
 #include "brave/components/ipfs/features.h"
 #endif
 
+#if !BUILDFLAG(IS_ANDROID)
```

Comment:
probably toolkit views is more appropriate here?

Reply:
actually even better would be to add a buildflag for this and set it to `enable_vertical_tabs = toolkit_views` in gn